SingularityUI/app/components/tasks/Columns.jsx

```diff
@@ -0,0 +1,302 @@
+import React, { PropTypes } from 'react';
+import Column from '../common/table/Column';
+
+import Utils from '../../utils';
+
+import JSONButton from '../common/JSONButton';
+import Glyphicon from '../common/atomicDisplayItems/Glyphicon';
+
+export const TaskId = (
+  <Column
+    label="Task ID"
+    id="taskId"
+    key="taskId"
```

Comment:
oh, interesting. What do you think about removing the `id` field from these and using `key` instead?